opentasks/src/main/java/com/jmedeisis/draglinearlayout/DragLinearLayout.java

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
Not our file, remove the copyright and license. This is the original license, please insert this one: https://github.com/justasm/DragLinearLayout/blob/master/LICENSE